docs/DeveloperGuide.md

```diff
@@ -654,8 +654,51 @@ Names in should only contain alphanumeric characters
 Example: listPerson Alex David
 ```
 
-### Improved Space Sensitivity in Names
+### Stronger Email Input Validation
+- **Background**: Currently, the `Email` of our `Person` is currently validated
+with the format of `local-part@domain`. The local-part and domain have their own
+restrictions with the aim of allowing SnS to reject invalid emails.
+
+- **Issue**: This validation does not strictly comply with the IETF standards for email
+addresses, resulting in invalid emails to be considered as valid by our application.
+For example, `a@12.34` has an invalid domain that does not comply with IETF standards
+but will be considered valid in our application.
```

Comment:
Can also add examples of overzealous validation. The local-part of emails can have special characters like / and = but we do not allow it.

Reply:
Added further elaboration